addon/utils/_private/rdfa-parser/rdfa-parser.ts

```diff
@@ -964,6 +964,8 @@ export class RdfaParser<N> {
     node: N,
     activeTag: IActiveTag<N>,
     attributes: Record<string, string>,
+    contentDatatype?: ModelNamedNode<N>,
```

Comment:
Are these necessary?

Reply:
yes, its how I distinguish between normal triples and contentliterals

Reply:
sorry that's actually not true, these indeed aren't necessary, it's only in the resourceNode function that they matter